deflate.c

```diff
@@ -190,8 +190,10 @@ int32_t ZNG_CONDEXPORT PREFIX(deflateInit2)(PREFIX3(stream) *strm, int32_t level
     deflate_state *s;
     int wrap = 1;
 
+#ifndef DISABLE_RUNTIME_CPU_DETECTION
     /* Force initialization functable, because deflate captures function pointers from functable. */
     functable.force_init();
+#endif
```

Comment:
**suggestion:** 
I am wondering whether it'd be cleaner and more in theme with the rest of it like `FUNCTABLE_CALL` etc if this instead was replaced by something like this:
```
    /* Force initialization functable, because deflate captures function pointers from functable. */
    FUNCTABLE_INIT;
```